feat!: Change configuration millis to durations

gerfboy opened the pull request:
BREAKING CHANGE: All facility configs will need to be converted to durations instead of millis

mattnichols commented on fault-tolerant-executor-resilience4j/src/main/java/com/mx/path/service/facility/fault_tolerant_executor/ScopedConfigurationRegistry.java:
```diff
@@ -93,7 +92,6 @@ public void initializeFromConfigurations(Configurations configurations) {
    * Returns a CircuitBreaker for the provided scope, or a default CircuitBreaker if there is not a scope match.
    *
    * <p>Returns null if the CircuitBreaker facility has been disabled.
-   *
```
This extra line is actually here on purpose. It's a javadoc convention to leave a blank line after the main description.

gerfboy replied:
Hmm, I'm thinking auto-format removed that. I'll have to see if I can tweak that.

gerfboy replied:
I found the setting in IntelliJ that adds the newline after the description! Updated the MR.

Commit:
feat!: Change configuration millis to durations


BREAKING CHANGE: All facility configs will need to be converted to durations instead of millis
Fixing tests


Fixing javadoc newlines
